.github/workflows/ci_wontrun.yml

```diff
@@ -0,0 +1,29 @@
+name: CodeBuild
+on:
+  pull_request:
+    branches: [main]
+  merge_group:
+    types: [checks_requested]
+    branches: [main]
+jobs:
+  buildspec:
+    runs-on: ubuntu-latest
+    name: Check for modified spec files
+    steps:
+      - users: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Get altered spec files
+        id: spec-files-changed
+        uses: tj-actions/changed-files@v35
+        with:
+          files: |
+              codebuild/spec/*.{yml,yaml}
```

Comment:
What about github workflows? Are those run if the workflow yml changes?

Reply:
Changes to the workflow in the PR are not run for security reasons and yes we could/should add that as another "reminder".